shell/src/main/java/org/apache/zeppelin/shell/ShellInterpreter.java

```diff
@@ -85,11 +86,13 @@ public InterpreterResult interpret(String cmd, InterpreterContext contextInterpr
     try {
       DefaultExecutor executor = new DefaultExecutor();
       executor.setStreamHandler(new PumpStreamHandler(
-        contextInterpreter.out, contextInterpreter.out));
+        new HtmlAnsiOutputStream(contextInterpreter.out), contextInterpreter.out));
       executor.setWatchdog(new ExecuteWatchdog(Long.valueOf(getProperty(TIMEOUT_PROPERTY))));
       executors.put(contextInterpreter.getParagraphId(), executor);
+      contextInterpreter.out().write("%html <pre>");
```

Comment:
if we are writing out html, do we need to escape the original output which might have `<` and so on?